### PR TITLE
App refactor

### PR DESCRIPTION
**Pull request by trentontri**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+server.listen(port);
+server.on('error', onError);
+server.on('listening', onListening);
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
```
Comment (chadwithuhc) [inline server/index.js]: you already have these two in there

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+});
+
+/* GET home page. */
+app.get('/', function (req, res, next) {
+  res.render('index', {title: 'Express'});
+});
+
+app.get("/api/:year/:race", function (req, res) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
```
Comment (chadwithuhc) [inline server/index.js]: `res.header()` will be setting things for the response of the call. Express should be handling most of this for you. any reason you need this here?

Comment (trentontri) [reply server/index.js]: @chadwithuhc I moved these into `server.js` is that what you were suggesting?
```
app.use(function(req, res, next) {
  res.header("Access-Control-Allow-Origin", "*");
  res.header("Access-Control-Allow-Credentials", true);
  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
  res.header("Access-Control-Allow-Headers", 'Origin,X-Requested-With,Content-Type,Accept,content-type,application/json');
  next();
});
```

File: server/index.js (hunk 1 above)
Comment (trentontri) [reply server/index.js]: @chadwithuhc noted! I refactored this under the server.js file

File: build/index.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+server.listen(port);
+server.on('error', onError);
+server.on('listening', onListening);
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
```
Comment (chadwithuhc) [inline build/index.js]: I still see you 👀

File: build/index.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
+var bodyParser = require("body-parser");
```
Comment (chadwithuhc) [inline build/index.js]: inconsistent quotes

File: build/index.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+});
+
+/* GET home page. */
+app.get('/', function (req, res, next) {
+  res.render('index', {title: 'Express'});
+});
+
+app.get("/api/:year/:race", function (req, res) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
```
Comment (chadwithuhc) [inline build/index.js]: inconsistent quotes

File: server/index.js (hunk 1 above)
Comment (chadwithuhc) [reply server/index.js]: oh, now I see there are two files. why is there two? delete the one you don't want.

File: server/index.js (hunk 2 above)
Comment (chadwithuhc) [reply server/index.js]: Yes, that is the basic cors middleware. why'd we need to move it?

File: build/index.js (hunk 3 above)
Comment (trentontri) [reply build/index.js]: @chadwithuhc I'm not sure what you're wanting me to do here? That's under the auto generated build folder. It get's copied from the root directory

Comment (chadwithuhc) [reply build/index.js]: You shouldn't need to build your server side app. It was confusing why there were two files with the same contents.

**Pull request merged**